custom_components/opnsense/sensor.py

```
@@ -374,21 +423,44 @@
+        return self._available
+
+    # Move this to OPNsenseEntity once all entity-types are updated
+    async def async_added_to_hass(self) -> None:
+        await super().async_added_to_hass()
+        self._handle_coordinator_update()
+
+
+class OPNsenseStaticKeySensor(OPNsenseSensor):
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
```

Ruff

`_handle_coordinator_update` is too complex (11 > 10)

(C901)